packages/fiori-freestyle-writer/package.json

```diff
@@ -32,6 +32,7 @@
         "!dist/**/*.map"
     ],
     "dependencies": {
+        "@sap-ux/eslint-plugin-fiori-tools": "workspace:*",
```

Comment:
That should also be a devDependency, correct?

Reply:
Thanks. Fixed in https://github.com/SAP/open-ux-tools/pull/984/commits/961518fbd58fe0074f714da7fcb906c19c4c9cc9